core/blockly.js

```diff
@@ -92,25 +93,12 @@ Blockly.selected = null;
 Blockly.draggingConnections = [];
 
 /**
- * Contents of the local clipboard.
- * @type {Element}
- * @private
- */
-Blockly.clipboardXml_ = null;
-
-/**
- * Source of the local clipboard.
- * @type {Blockly.WorkspaceSvg}
- * @private
+ * Get the current contents of the clipboard and associated metadata.
+ * @return {{xml: ?Element, source: ?Blockly.WorkspaceSvg, typeCounts: ?Object}}
+ *     An object containing the clipboard contents and associated metadata.
+ * @public
  */
-Blockly.clipboardSource_ = null;
-
-/**
- * Map of types to type counts for the clipboard object and descendants.
- * @type {Object}
- * @private
- */
-Blockly.clipboardTypeCounts_ = null;
+Blockly.getClipboardInfo = Blockly.clipboard.getClipboardInfo;
```

Comment:
Do we need to add this here since it is a new API?

Reply:
I removed this for now. However, we will have to revisit when we remove `declareLegacyNamespace` in `clipboard.js`. Since the clipboard didn't exist at all before this quarter, and I prefer to keep it internal, I may end up adding this single function at the top level.